### ISSUE TITLE
Performance drop when using DDP

### ISSUE THREAD

**Issue by lzongwei** [zdou0830/METER#13]
Hi! Thanks for your great job! i tried to pretrain the model on multi node,multi gpus (8*8 gpus as vilt did), and observed a performance drop when finetune in VQA. Is there any config difference between pre-training in single node, 1*8 gpus as readme recommended, and multi node(n*8 gpus) ?

**Comment by zdou0830** [comment]
Hi, the config settings should be the same. Here is the file that I used for distributed training https://github.com/zdou0830/METER/blob/main/azure_distributed_run.py, not sure if it helps.

**Comment by lzongwei** [comment]
i tried to run**python run.py with data_root=DATA_PATH num_gpus=8 num_nodes=1 task_mlm_itm_clip_bert per_gpu_batchsize=32 clip16 text_roberta image_size=288** but a GPU OOM occured. i use the 40GB A100 as recommended. Have you ever met this? i tried to set the per_gpu_batchsize=16 to avoid oom and keep the total batch size 4096.  Will this cause a performance drop?

**Comment by zdou0830** [comment]
Hello, reducing the per_gpu_batchsize will not cause a performance drop as the gradients will accumulate and the total batch size remains unchanged.

**Comment by lzongwei** [comment]
Hi Ziyi. I still can not reproduce the VQA result when pretraining on my own, while loading the pretrained checkpoint you provided can. I followed all the recommended configs and did not change any of the code. Could you please provide the pretrain data arrow and the tensorboard file? Thanks again for your patience!

**Comment by mactavish91** [comment]
> Hi! Thanks for your great job! i tried to pretrain the model on multi node,multi gpus (8 * 8 gpus as vilt did), and observed a performance drop when finetune in VQA. Is there any config difference between pre-training in single node, 1 * 8 gpus as readme recommended, and multi node(n*8 gpus) ?

@lzongwei Hello, have you solved the problems you encountered before?
